autoemulate/experimental/tuner.py

```diff
@@ -24,20 +31,18 @@ class Tuner:
         The validation scores and parameter values used in each search iteration.
     """
 
-    def __init__(self, dataset: Dataset, n_iter: int):
+    def __init__(self, x: InputLike, y: OutputLike | None, n_iter: int):
```

Comment:
Should y be InputLike here insteadof OutputLike?

Reply:
Thanks - yes agreed!